setup-scripts/setup-linux-desktop.bash

```diff
@@ -44,8 +44,7 @@ mamba install -c conda-forge --yes \
       jupyter-server-proxy \
       nbgitpuller
 
-# Until https://github.com/jupyterhub/jupyter-remote-desktop-proxy/issues/87 is sorted
-python -m pip install --no-cache git+https://github.com/jupyterhub/jupyter-remote-desktop-proxy@a7c1ae9bca94ca3a4bbf90f7a105e16b1603d4bd
+python -m pip install --no-cache "jupyter-remote-desktop-proxy==2.*"
```

Comment:
I suggest pinning only to the major version, as that seems like a good compromise between stability and maintenance burden for this manually updated dependency.